ambry-store/src/test/java/com/github/ambry/store/IndexTest.java

```diff
@@ -1157,6 +1162,167 @@ public void tokenRevalidationTest() throws StoreException {
     }
   }
 
+  @Test
+  public void rebuildTokenWhenLastSegmentAutoClosedTestCase1() throws StoreException {
```

Comment:
this test case has nothing to do with auto-close last segment? Am I missing anyting?